src/runtime_src/core/edge/common/aie_parser.cpp

```diff
@@ -329,16 +397,16 @@ get_trace_gmio(const pt::ptree& aie_meta)
 
 namespace xrt_core { namespace edge { namespace aie {
 
-adf::driver_config
-get_driver_config(const xrt_core::device* device)
+std::vector<std::string>
```

Comment:
I think this probably revert what @jerryhsu-xilinx has done for common layer code. You need to talk to him about it. Looks to me, this will break the graph control path to use the common code between xrt and adf

Reply:
We had merged changes, and it looked confusing. I re-arranged the files so our changes look cleaner. Thanks for the heads up.